awx/api/templates/api/api_o_auth_authorization_root_view.md

```diff
@@ -8,7 +8,28 @@ reference. The `implicit` grant type can only be used to acquire a access token
 AWX context (Note AWX net location default to `http://localhost:8013` in examples):
 
 
-## Authorization using application of grant type `implicit`
+## Application using `authorization code` grant type
+This application grant type is intended to be used when the application is executing on the server.  To create
+an application named `AuthCodeApp` with the `authorization-code` grant type, 
+Make a POST to the `/api/v2/applications/` endpoint.
+```text
+{
+    "name": "AuthCodeApp",
+    "user": 1,
+    "client_type": "confidential",
+    "redirect_uris": "http://localhost:8013/api/v2",
+    "authorization_grant_type": "authorization-code",
+    "skip_authorization": false
+}
+```
+You can test the authorization flow out with this new application by copying the client_id and URI link into the 
+homepage here (http://django-oauth-toolkit.herokuapp.com/consumer/) and click submit. This is just a simple test 
+application Django-oauth-toolkit provides. 
```

Comment:
I wonder if maybe this is a little chatty to be in the API help text itself. This would be nice in the acceptance docs, but these templates go out in the OPTIONS and we should be very dry when writing them.